mlir/lib/Bindings/Python/IRModule.h

```diff
@@ -841,6 +841,8 @@ class PyInsertionPoint {
   PyInsertionPoint(const PyBlock &block);
   /// Creates an insertion point positioned before a reference operation.
   PyInsertionPoint(PyOperationBase &beforeOperationBase);
+  /// Creates an insertion point positioned before a reference operation.
+  PyInsertionPoint(PyOperationRef beforeOperationRef);
```

Comment:
do you really need this? can't you just do `PyOperation::forOperation(ctx, op)->getOperation()` and use the above one?

Reply:
fair enough